en/docs/setup/analytics/configure-analytics-for-elk-stack.md

```diff
@@ -70,8 +70,47 @@ You can also set multiple publishers for analytics as follows. Replace ```choreo
 
 To configure Elasticsearch and Kibana on your Kubernetes cluster, you can refer to the [official guide](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-stack-helm-chart.html) provided by Elastic. The guide includes instructions on deploying the necessary Helm charts for Elasticsearch and Kibana.
 
+You can also install Elasticsearch and Kibana using the direct helm charts for each. The following is a sample setup that can be used to set up both. However, note that these are only for testing, and must be configured as necessary for a production environment.
 
-## Step 3 - Setup Logging agent 
+### Installing Elasticsearch
+
+1. Apply the following commands to install Elasticsearch.
+
+```bash
+helm repo add elastic https://helm.elastic.co
+helm repo update
+helm install elasticsearch elastic/elasticsearch
+```
+2. Wait for the pods to spin up. You can view them using the following command.
+```bash
+kubectl get pods
+```
+3. Once the pods all have spun up, you can get the elasticsearch credentials using the following command.
```

Comment:
Add a blank line before and after a bullet point to increase readability. Same for bullet no.4 also.

Reply:
Fixed in PR #583